modules/045-snapshot-controller/templates/snapshot-controller/deployment.yaml

```diff
@@ -0,0 +1,72 @@
+{{- if (.Values.global.enabledModules | has "vertical-pod-autoscaler-crd") }}
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  name: d8-snapshot-controller
+  namespace: kube-system
+  {{- include "helm_lib_module_labels" (list . (dict "app" "d8-snapshot-controller" "workload-resource-policy.deckhouse.io" "master")) | nindent 2 }}
+spec:
+  targetRef:
+    apiVersion: "apps/v1"
+    kind: Deployment
+    name: d8-snapshot-controller
+  updatePolicy:
+    updateMode: "Auto"
+{{- end }}
+---
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  annotations:
+    helm.sh/hook: post-upgrade, post-install
```

Comment:
What?

Reply:
This was introduced in all modules by https://github.com/deckhouse/deckhouse/commit/6fb39beb1e45a2668f98cbd87afb75c62affff53.

I'm assuming the PDB violates the deploying process in some cases when it is processed in the standard way.

Reply:
I can't remember why :(.

Reply:
Ok, leave it be then.